views/layouts/MainLayout.jsx

```diff
@@ -7,46 +7,81 @@ import Sidebar from '../components/Sidebar'
 import api from '../api'
 import getCounts from '../fe-utils/countUtil'
 import { fetchSubjects } from '../fe-utils/fetchUtil'
-import { AuthContext, NotificationContext } from '../contexts'
+import {
+  AuthContext,
+  ConfigurationsContext,
+  NotificationContext,
+  SidebarContext,
+} from '../contexts'
 import { headerTitle } from './helpers'
 
 const MainLayout = ({ classes, theme }) => {
+  const temporary = 'temporary'
+  const persistent = 'persistent'
+  const dashboard = 'dashboard'
```

Comment:
let's extract these outside the component so we're no re-declaring them with every render.

Reply:
Let's also be a little more specific with the naming, like `TEMPORARY_SIDEBAR`, `PERSISTENT_SIDEBAR`, etc.

Reply:
[2c26c5e](https://github.com/AMP-SCZ/dpdash/pull/352/commits/2c26c5ef82ff63420783fb443c4cd7a14df00535)